custom_reg_form/admin.py

```diff
@@ -1,4 +1,41 @@
+from typing import Type
+
 from django.contrib import admin
+from django.contrib.auth import get_user_model
+
 from .models import ExtraInfo
 
-admin.site.register(ExtraInfo)
+User = get_user_model()
+
+
+class BaseAdmin(admin.ModelAdmin):
+    """Base class that allow to extract username from user field.
+
+    methods:
+        get_author_username: Returns username from User instance.
+    """
+
+    @admin.display(ordering="user__username", description="User")
+    def get_user_username(self, obj: Type[User]) -> str:
+        """Return username from User instance"""
+        return obj.user.username
+
+
+class ExtraInfoAdmin(BaseAdmin):
+    """Admin class for ExtraInfoAdmin.
+
+    attributes:
+        list_display: Fields to be shown in admin interface.
+        search_fields: fields that use to search and find matches.
+    """
+
+    list_display = (
+        "get_user_username",
+        "arabic_name",
+        "allow_newsletter_emails",
+    )
+    search_fields = ("user__id", "user__username", "arabic_name")
```

Comment:
For the user is enough username.
```suggestion
    search_fields = ("user__username", "arabic_name")
```